### PR TITLE
[CIVIS-9031] Deprecate `metrics` cmd in favour of `measures` cmd

### PR DESCRIPTION
**Pull request by ManuelPalenzuelaDD**
This patch renames the `metrics` command to `measures`. We'll still support the `metrics` command name but it'll print a deprecation warning when used. Additionally, we do the same change to the `--metrics` option of the command (renaming it to `--measures`)

Context: https://docs.google.com/document/d/1X9-0eiiNaDsfLMKlVYsGdQs4RDKGZx9jEqLX4NEPAxY

This is a sister pr of https://github.com/DataDog/datadog-ci/pull/1203

- [x] Feature or bugfix MUST have appropriate tests (unit, integration)

### CODE REVIEW

File: src/commands/measure/README.md
Comment (juan-fernandez) [inline src/commands/measure/README.md:14]:
```suggestion
datadog-ci measure --level job --measures binary.size:1024
```

File: README.md
Comment (juan-fernandez) [inline README.md:71]: is the idea to update the documentation too? We refer to metrics still. We might want to reflect that this link is not updated somewhere

**Review by juan-fernandez — APPROVED** [review]

File: README.md
Comment (ManuelPalenzuelaDD) [reply README.md:71]: Yes, I'm currently changing the docs too (https://github.com/DataDog/documentation/pull/21897).
I will rename the mentions to the outdated docs as soon as that is merged.

File: src/commands/measure/README.md
Comment (ManuelPalenzuelaDD) [reply src/commands/measure/README.md]: Nice catch, thank you!

File: src/commands/measure/README.md
Comment (aliciascott) [inline src/commands/measure/README.md:58]:
```suggestion
[1]: https://docs.datadoghq.com/continuous_integration/pipelines/custom_tags_and_metrics/
```

Just noting this link will need changed to the new page

**Review by aliciascott — APPROVED** [review]
Approved; pending changing the link to the newly named page as well.

**Pull request merged**
